Fabric.Authorization.AccessControl/src/app/services/fabric-auth-group.service.ts

```diff
@@ -67,11 +67,21 @@ export class FabricAuthGroupService extends FabricBaseService {
       .post<Group>(this.replaceGroupNameSegment(FabricAuthGroupService.groupRolesApiUrl, groupName), role);
   }
 
```

Comment:
I think we need to remove the method above (`addRoleToGroup`) since that's no longer supported.

Reply:
yeah, i will do that